### PR TITLE
feat(`st2`): add completion

### PR DESCRIPTION
**Pull request by mamercad**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

New completion

**What is the current behavior? (You can also link to an open issue here)**

There is no completion for `st2`

**What is the new behavior (if this is a feature change)?**

There will be completion for `st2`

**Additional info:**

[st2client](https://pypi.org/project/st2client/)

### CODE REVIEW

File: src/st2.ts [hunk 1]
```diff
@@ -0,0 +1,2423 @@
+// NOTE: Copied this generator from chown.ts
```
Comment (fedeci) [inline src/st2.ts:1]: Is there any way to get this shared between the two tools? Maybe just a variable export?

**Review by fedeci — APPROVED** [review]
Great work as always! @SeparateRecords if you want to give another look at this before merging it may help to catch eventual errors I missed.

File: src/st2.ts (hunk 1 above)
Comment (mamercad) [reply src/st2.ts:1]: I'll look into it when I get some cycles!

Comment (mamercad) [reply src/st2.ts]: @fedeci I'm not well-versed in Typescript -- how's this look?

Comment (caronmarc033) [reply src/st2.ts]: > Is there any way to get this shared between the two tools? Maybe just a variable export?

**Comment by SeparateRecords** [comment]
LGTM! This is awesome @mamercad! Linter is failing because of `ng`, fixed in master branch so not an issue.

Absolutely incredible that you also wrote this whole thing by hand. There's no good way to automate it because it's built with Python's builtin, highly imperative `argparse` module. Would love to give you a shoutout on Twitter from [@fig](twitter.com/fig) if you're open to it!

**Review by SeparateRecords — APPROVED** [review]

**Comment by fedeci** [comment]
@SeparateRecords FYI we now have an argparse integration!

**Comment by SeparateRecords** [comment]
@fedeci no way!! Totally missed that this was added along with the cement completions. Super cool.

**Comment by mamercad** [comment]
> LGTM! This is awesome @mamercad! Linter is failing because of `ng`, fixed in master branch so not an issue.
> 
> Absolutely incredible that you also wrote this whole thing by hand. There's no good way to automate it because it's built with Python's builtin, highly imperative `argparse` module. Would love to give you a shoutout on Twitter from [@fig](https://twitter.com/fig) if you're open to it!

Ship it! 😄

**Pull request merged**